### PR TITLE
Dashboard 2 cleanup

### PR DESCRIPTION
**Pull request by tmitchellgcio**
## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by erikphansen — CHANGES_REQUESTED** [review]

File: src/platform/site-wide/mega-menu/containers/Main.jsx [hunk 1]
```diff
@@ -123,13 +121,12 @@ export class Main extends Component {
...
 const mapStateToProps = (state, ownProps) => {
   const loggedIn = isLoggedIn(state);
-  const showDashboard2 = selectShowDashboard2(state);
 
   // Derive the default mega menu links (both auth + unauth).
   const defaultLinks = ownProps?.megaMenuData ? [...ownProps.megaMenuData] : [];
 
   // Add the My VA link to default links if we are showing dashboard 2 or if we are logged in.
-  if (showDashboard2 || loggedIn) {
+  if (loggedIn) {
```
[the remark this answers is not in the document]
Comment (tmitchellgcio) [reply src/platform/site-wide/mega-menu/containers/Main.jsx:129]: Should we not so a check for logged in though? I assumed this had a purpose outside of the dashboard2

[the remark this answers is not in the document]
Comment (erikphansen) [reply src/platform/site-wide/mega-menu/containers/Main.jsx:129]: I think @kelsonic or someone on his team would know better (and I think his team has to approve this change anyway), but the idea was that, with the launch of My VA 2.0 to all users, we will always show the My VA link on the main nav, whether you are signed in or not.

[the remark this answers is not in the document]
Comment (kelsonic) [reply src/platform/site-wide/mega-menu/containers/Main.jsx:129]: Yupp! That's correct, you can omit the `if (loggedIn) {` bit assuming we want the My VA link to always show regardless of auth 🙂 Thanks @erikphansen!

[the remark this answers is not in the document]
Comment (tmitchellgcio) [reply src/platform/site-wide/mega-menu/containers/Main.jsx]: removed

**Review by erikphansen — APPROVED** [review]
Looks good. I also pulled this down locally and made sure the post-Dashboard-2 world looked good. This PR is mostly moving existing files and then updating some imports. I'm confident enough in the existing test coverage that nothing broke in this reorg. Nice job!

**Review by kelsonic — APPROVED** [review]

**Review by alexandec — APPROVED** [review]

**Comment by erikphansen** [comment]
@tmitchellgcio I can't make sense of the Cypress failures on Jenkins :-/ I ran all of the Dashboard Cypress tests locally and it's fine. Running the entire suite locally and it fails and exits early. Can you maybe try merging in `master` yet again to see if that fixes things?

**Review by micahchiang — APPROVED** [review]

**Comment by tmitchellgcio** [comment]
@department-of-veterans-affairs/chatbot-admin could I get a review?

**Review by jgacek — APPROVED** [review]

**Pull request merged**